.buildkite/pipeline.yml

```diff
@@ -21,7 +21,10 @@ steps:
       system: x86_64-linux
 
   - label: 'Check Cabal Configure'
-    command: 'nix-shell --run "cabal v2-update && cabal v2-configure --enable-tests --enable-benchmarks"'
+    command:
+      - 'nix-shell --run "cabal v2-update"'
+      - 'nix-shell --run "cabal v2-configure --enable-tests --enable-benchmarks"'
+      - 'nix-shell --run "cabal v2-configure -frelease --enable-tests --enable-benchmarks"'
```

Comment:
Running it twice, one without and once with the flag? Is this intended?

Reply:
It was intended. In case someone puts something silly under `if (!flag(release))`